ChangeLog.d/fix-rsa-performance-regression.txt

```diff
@@ -0,0 +1,2 @@
+Bugfix
+   * Fix unintended performance regression when using short RSA public keys. Fixes #9232.
```

Comment:
Is this line going to be too long?

Reply:
It is within the 100 column limit. Do we have a stricter limitation on Changelog entries?

Reply:
```
$ scripts/assemble_changelog.py
Traceback (most recent call last):
  File "/media/psf/Home/home/tom/projects/mbedtls/quick/scripts/assemble_changelog.py", line 534, in <module>
    main()
  File "/media/psf/Home/home/tom/projects/mbedtls/quick/scripts/assemble_changelog.py", line 531, in main
    merge_entries(options)
  File "/media/psf/Home/home/tom/projects/mbedtls/quick/scripts/assemble_changelog.py", line 477, in merge_entries
    changelog.add_file(input_file)
  File "/media/psf/Home/home/tom/projects/mbedtls/quick/scripts/assemble_changelog.py", line 257, in add_file
    self.add_categories_from_text(input_stream.name, 1,
  File "/media/psf/Home/home/tom/projects/mbedtls/quick/scripts/assemble_changelog.py", line 225, in add_categories_from_text
    raise InputFormatError(filename,
__main__.InputFormatError: ChangeLog.d/fix-rsa-performance-regression.txt:2: Line is longer than allowed: Length 89 (Max 80)
```